addons/payment_stripe/views/payment_templates.xml

```diff
@@ -31,4 +19,16 @@
         <script type="text/javascript" src="https://js.stripe.com/v3/"></script>
     </template>
 
+    <template id="payment_adyen.method_form" inherit_id="payment.method_form">
```

Comment:
Maybe it's a typing error, maybe it's currently the only way to make it works ^^
```suggestion
    <template id="payment_stripe.method_form" inherit_id="payment.method_form">
```